ROS/Chapter_5.ipynb

```diff
@@ -0,0 +1,505 @@
+{
```

Comment:
<p>use "constrained_layout=True"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>This constrained layout does not look very good... </p>

Reply:
![image](https://user-images.githubusercontent.com/7213793/87115493-59d78b80-c228-11ea-8f6f-5a3847fad017.png)

Reply:
Nevermind fixed it with some line breaks 
![image](https://user-images.githubusercontent.com/7213793/87115568-84294900-c228-11ea-97b2-2949bbd3c270.png)